scripts/main.coffee

```diff
@@ -13,13 +13,14 @@ module.exports = (robot) ->
 
   config = require('../config.json')
   messages = require('./messages.js').messages
+  formatting = require('./formatting.js').formatting
 
   # messages.setAuth((err, res) ->
   #     console.log("Logged in: " + res[1].title)
   # )
 
   robot.hear /.*/i, (res) ->
-      message = messages.getAction(res.message.text)
+      message = formatting.getDataMask(res.message.text, /\+[^*\s]+/)
```

Comment:
Ok, but why it is the part of this commit? i do not see any changes to `messages` in this commit. 

I would do this in separate commit with reasonable message Eg "moved part of methods from messages, that are just reffer to formatting messages, not to just sending them"